manifest.json

```
@@ -1,7 +1,7 @@
 {
   "name": "Nabu Casa Zendesk Theme",
   "author": "Nabu Casa",
-  "version": "1.0.32",
+  "version": "1.0.31",
```

```suggestion
  "version": "1.0.33",
```
Not sure if this can go back, better to move forward.